Analysis/include/AnaTuple.h

```diff
@@ -27,33 +27,59 @@ namespace analysis {
 #define TAU_DATA(name) \
     P4_DATA(name) \
     VAR_LIST(float, name##_iso, name##_DeepTauVSe, name##_DeepTauVSmu, name##_DeepTauVSjet) \
-    VAR_LIST(int, name##_q, name##_gen_match) \
+    VAR_LIST(int, name##_q, name##_gen_match, name##_decay_mode) \
     /**/
 
 #define JET_DATA(name) \
     P4_DATA(name) \
-    VAR_LIST(float, name##_CSV, name##_DeepCSV, name##_DeepFlavour, name##_DeepFlavour_CvsL, name##_DeepFlavour_CvsB, \
+    VAR_LIST(float, name##_DeepFlavour, name##_DeepFlavour_CvsL, name##_DeepFlavour_CvsB, \
                     name##_HHbtag) \
     VAR_LIST(int, name##_valid, name##_hadronFlavour) \
 
+#define FAT_JET_DATA(name) \
+    P4_DATA(name) \
+    VAR(float, name##_m_softDrop) \
+    VAR(int, name##_valid) \
+
 #define ANA_EVENT_DATA() \
     VAR(std::vector<size_t>, dataIds) /* EventAnalyzerDataId */ \
     VAR(std::vector<double>, all_weights) /* all weight */ \
     VAR(std::vector<float>, all_mva_scores) /* all mva scores */ \
+    VAR(std::vector<float>, btag_weight_loose) /* btag weight Loose wp: up,central,down */ \
```

Comment:
to simplify how the code to access btag weights in ProcessAnaTuple:
`btag_weight_loose` -> `btag_weight_Loose` etc. (like this `"btag_weight_" + ToString(wp)` will work).
- update comment: now they are stored in different order: central,up,down